xup/xup.c

```diff
@@ -212,7 +212,7 @@ lib_mod_connect(struct mod *mod)
         if (trans_connect(mod->trans, mod->ip, con_port, 3000) == 0)
         {
             LLOGLN(0, ("lib_mod_connect: connected to Xserver "
-                   "(Xorg or X11rdp) sck %ld", mod->trans->sck));
+                   "(Xorg or X11rdp) sck %ld", (long) (mod->trans->sck)));
```

Comment:
Then let's use `long long` here to be sure were are not cutting away the top half of the pointer on Windows. `%p` won't be good, as sockets are not generally printed in hexadecimals on Linux.